distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -62,7 +65,7 @@ type Function
    > Example
      five = Function.identity 5 # returns number 5
 
-# identity : a -> a
+   ## identity : a -> a
 identity : Any -> Any
 identity x = x
```

Comment:
What happened here??

The indentation level seems to be all off.

Besides, I think this method should have a proper doc comment on it.

Reply:
There is - there are 2 comments above these functions at the moment. The commented out typedef line and the doc comment. Change here was just to move these commented typedefs into the doc comment.

Reply:
Sorry indeed I misread it. I'd just remove these 'fake' sigs.